### PR TITLE
MSBuild: Document VerifyFileHash task

### PR DESCRIPTION
**Pull request by rainersigwald**
cc @natemcmaster (contributor of these tasks) and @japj (who asked for these docs in https://github.com/Microsoft/msbuild/pull/3999#issuecomment-458141311).

### CODE REVIEW

**Comment by PRMerger10** [comment]
@rainersigwald : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

File: docs/msbuild/getfilehash-task.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  - "http://schemas.microsoft.com/developer/msbuild/2003#GetFileHash"
+dev_langs:
+  - "VB"
+  - "CSharp"
+  - "C++"
+  - "jsharp"
+helpviewer_keywords:
+  - "GetFileHash task [MSBuild]"
+  - "MSBuild, GetFileHash task"
+ms.assetid: ???
```
Comment (rainersigwald) [inline docs/msbuild/getfilehash-task.md]: @Mikejo5000 should I just create a new GUID for these, or is there something else I could do to help? In general I was guessing on a lot of this metadata.

Comment (Mikejo5000) [reply docs/msbuild/getfilehash-task.md]: @rainersigwald ms.assetid is used only for topics migrated from MSDN, so please remove that. Everything else looks good except f1_keywords. This metadata item is for F1 links from the IDE (if an F1 link exists) formatted like this example: vc.task.xdcmake
If there's no plan to link to the topic from the IDE, just remove f1_keywords.
Thanks!

**Comment by ktoliver** [comment]
Hi @rainersigwald 

Microsoft authors and contributors need to make content contributions through the private repository so the content can be staged and validated by the current validation rules. 

After you review staging and fix all validation issues, you can add the sign-off comment to let PR reviewers know that the updates are ready to be merged. 

For more information, see [Should I work in a private or a public repo?](https://review.docs.microsoft.com/en-us/help/contribute/contribute-authors-in-public-repo?branch=master) 

If you need help getting set up to work in the private repo, see [Set up Git repository locally for documentation](https://review.docs.microsoft.com/en-us/help/contribute/contribute-get-started-setup-local?branch=master).

Would you create a pull request in the private repo to make these updates? We'll close this PR. Thank you.


CC @Mikejo5000 @natemcmaster

**Comment by natemcmaster** [comment]
That's a really confusing workflow.

**Pull request closed without merging**